src/pages/api/bookings/POST.ts

```diff
@@ -0,0 +1,19 @@
+import connectMongo from "../../../util/connectMongo";
+import Booking from "../../../../models/booking";
+
+/**
+ * @param {import('next').NextApiRequest} req
+ * @param {import('next').NextApiResponse} res
+ */
+
+export default async function createBooking(req: any, res: any) {
+  try {
+    await connectMongo();
+    const booking = await Booking.create(req.body);
+    res.json({ booking: booking });
+    
+  } catch (error) {
+    console.log(error);
+    res.json({ error });
```

Comment:
It's generally not a great idea to include the exception you've caught in the public response as is, because you're not in control of what information it contains and some of it may be harmful. I would recommend manually picking the information you need from the exception and include that in your API errors.